test/infrastructure/docker/templates/cluster-template-development.yaml

```diff
@@ -78,6 +66,18 @@ spec:
 ---
 apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
 kind: DockerMachineTemplate
+metadata:
```

Comment:
Just moved this down as it's the more logical order and to make it easy to diff against (and thus keep in sync with)  the ClusterClass